pom.xml

```diff
@@ -162,16 +169,16 @@
     <security.saml.sloUrl></security.saml.sloUrl>
     <security.saml.maximumAuthenticationLifetime>60</security.saml.maximumAuthenticationLifetime>
 
-    <security.auth.windows.enabled>false</security.auth.windows.enabled>
-    <security.auth.kerberos.enabled>false</security.auth.kerberos.enabled>
+    <security.auth.windows.enabled>true</security.auth.windows.enabled>
```

Comment:
set all these 
security.auth.xxx to false
except security.auth.openid.enabled=true